python/GafferCortexUI/ParameterisedHolderUI.py

```diff
@@ -66,14 +70,21 @@ def __init__( self, node, readOnly=False, **kw ) :
 		column = GafferUI.ListContainer( GafferUI.ListContainer.Orientation.Vertical, spacing = 4 )
 
 		GafferUI.NodeUI.__init__( self, node, column, **kw )
+		
+		showHeader = True
+		parameterised = self.node().getParameterised()[0]
+		if isinstance( parameterised, IECore.Op ) :
```

Comment:
Any particular reason this is limited to Ops? Since the default is to keep the old behaviour, it seems to me we may as well support this for all Parameterised types.